src/status_im/ui/screens/pairing/styles.cljs

```diff
@@ -1,27 +1,78 @@
 (ns status-im.ui.screens.pairing.styles
```

Comment:
could you please fix styles for this file, map values alignments

Reply:
done!